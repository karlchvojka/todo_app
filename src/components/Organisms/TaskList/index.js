import React from 'react';
import './index.scss';

const TaskList = (props) => {
  return (
    <section className="taskList">
    </section>
  )
}

export default TaskList
