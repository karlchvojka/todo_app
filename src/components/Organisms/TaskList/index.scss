.taskList {
  box-shadow: 2px 1px 8px 5px rgba(48,191,242,0.75);
  height:200px;
  width:100%;
}
