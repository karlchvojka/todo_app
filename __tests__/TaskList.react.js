import React from 'react';
import {render} from '@testing-library/react';
import { create, renderer } from 'react-test-renderer';

import TaskList from '../src/components/Organisms/TaskList';

describe("The Task List component", () => {
  it('should render properly', () => {
    const tasklist = create(<TaskList />);
    expect(tasklist.toJSON()).toMatchSnapshot();
  })
});
