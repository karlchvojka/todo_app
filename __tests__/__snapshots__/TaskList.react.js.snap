// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Task List component should render properly 1`] = `
<section
  className="taskList"
/>
`;
